package/MDAnalysis/analysis/hole2/__init__.py

```diff
@@ -36,14 +36,23 @@
 
 """
 import warnings
-from mdahole2.analysis.hole import hole, HoleAnalysis
-from mdahole2.analysis import utils, templates
-from mdahole2.analysis.utils import create_vmd_surface
 
-wmsg = (
-    "Deprecated in version 2.8.0\n"
-    "MDAnalysis.analysis.hole2 is deprecated in favour of the "
-    "MDAKit madahole2 (https://www.mdanalysis.org/mdahole2/) "
-    "and will be removed in MDAnalysis version 3.0.0"
-)
-warnings.warn(wmsg, category=DeprecationWarning)
+try:
```

Comment:
I'm not sure why these are reporting as not covered - the tests for both warnings seem to be passing.

Reply:
I manually tested everything locally (with and without the optional imports) so I'd say it's probably worth ignoring codecov here.